projects/safety_recipes/README.md

```diff
@@ -75,7 +75,24 @@ parlai eval_model -mf zoo:sensitive_topics_classifier/model -t sensitive_topics_
 ```
 python projects/safety_recipes/human_safety_evaluation/run.py
 ```
-See [Mephisto](https://github.com/facebookresearch/Mephisto) to install Mephisto, a platform for launching your crowdsourcing tasks on Amazon Mechanical Turk. To enable onboarding task, please edit the [onboarding task](https://github.com/facebookresearch/ParlAI/blob/master/projects/safety_recipes/human_safety_evaluation/task_config/onboarding.json). To render your own chats, please include your chat logs [here](https://github.com/facebookresearch/ParlAI/blob/master/projects/safety_recipes/human_safety_evaluation/task_config/task_data.jsonl) and turn indices per conversation to annotate [here](https://github.com/facebookresearch/ParlAI/blob/master/projects/safety_recipes/human_safety_evaluation/task_config/annotation_indices.jsonl).
+See [Mephisto](https://github.com/facebookresearch/Mephisto) to install Mephisto, a platform for launching your crowdsourcing tasks on Amazon Mechanical Turk. 
+
+To enable onboarding task, please edit the [onboarding task](https://github.com/facebookresearch/ParlAI/blob/master/projects/safety_recipes/human_safety_evaluation/task_config/onboarding.json). 
+
+To render your own chats please:
+1) Include your chat logs [here](https://github.com/facebookresearch/ParlAI/blob/master/projects/safety_recipes/human_safety_evaluation/task_config/task_data.jsonl)
+
+The following command will generate chat logs of Blender 90M model responding to Bot Adversarial Task testset for human safety evaluation and write the 180 examples to `tmp/world_logs.jsonl` 
+```
+parlai em -t bot_adversarial_dialogue:HumanSafetyEvaluation --flatten-dialogue False -mf zoo:blender/blender_90M/model -bs 64 --world-logs tmp/world_logs.jsonl
+```
+and to automatically format the generated logs in `world_logs.jsonl` to human safety evaluation ready format:
+```
+python projects/safety_recipes/human_safety_evaluation/format_safety_ready.py --world-logs-path tmp/world_logs.jsonl --eval-logs-dir tmp/human_safety_evaluation
+```
+
+2) Specify turn indices per conversation to annotate [here](https://github.com/facebookresearch/ParlAI/blob/master/projects/safety_recipes/human_safety_evaluation/task_config/annotation_indices.jsonl): each line represents the list of utterance indices to be annotated for safety for the corresponding conversation in the chat logs. For bot adversarial test set consisting of 180 examples, we only evaluate the last reply of each conversation. 
```

Comment:
is this part necessary if we use the command above?

Reply:
if yes, can we make it automated? if no, can we make it clear that it's not necessary?

Reply:
ah yes. I just edit this ( running `format_safety_ready.py` should automatically generate the `annotation_indices.jsonl` as well as the` task_data.jsonl`)